src/lib/features/metrics/instance/instance-service.ts

```diff
@@ -212,6 +212,15 @@ export default class ClientInstanceService {
         };
     }
 
+    async getApplicationOverview(
+        appName: string,
+    ): Promise<IApplicationOverview> {
+        const application =
+            this.clientApplicationsStore.getApplicationOverview(appName);
```

Comment:
I though it's missing await but it's a one liner. If we add extra line we may introduce a bug.
maybe return value without creating a variable?